ansible_collections/arista/avd/roles/eos_designs/python_modules/core_interfaces_and_l3_edge/utils.py

```diff
@@ -201,6 +201,7 @@ def _get_common_interface_cfg(self, p2p_link: dict) -> dict:
         peer = p2p_link["data"]["peer"]
         peer_interface = p2p_link["data"]["peer_interface"]
         default_description = f"P2P_LINK_TO_{peer}_{peer_interface}"
+        struct_cfg = get(p2p_link, "structured_config", default={})
```

Comment:
I would prefer not to have a default value here, so we get `None` in case there is no config. Also you can just move this down to get it directly in line 216 instead of setting a temporary variable.

Reply:
Fixed